dags/deletes.py

```diff
@@ -22,29 +20,11 @@
     Mutation,
     MutationRunner,
     NodeRole,
-    get_cluster,
 )
 from posthog.models.event.sql import EVENTS_DATA_TABLE
 from posthog.models.async_deletion import AsyncDeletion, DeletionType
 
 
-class ClickhouseClusterResource(ConfigurableResource):
-    """
-    The ClickHouse cluster used to run the job.
-    """
-
-    client_settings: dict[str, str] = {
-        "max_execution_time": "0",
-        "max_memory_usage": "0",
-        "receive_timeout": f"{24 * 60 * 60}",  # wait 24 hours for a response from CH
```

Comment:
At a quick skim I don't see anything that would require such a long socket timeout at this point, but I may have missed something